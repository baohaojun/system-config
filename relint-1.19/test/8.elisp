;;; Relint test file 8          -*- emacs-lisp -*-

(defun test-cl-flet ()
  (looking-at
   (cl-flet ((f (x) (concat "+" x)))    ; "+A"
     (f "A"))))

(defun test-cl-flet* ()
  (looking-at
   (cl-flet* ((f (x) (concat "+" x)))   ; "+B"
     (f "B"))))

(defun test-cl-labels ()
  (looking-at
   (cl-labels ((f (x) (concat "*" x)))  ; "*C"
     (f "C"))))
