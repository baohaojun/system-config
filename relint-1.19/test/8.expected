8.elisp:5:4: In call to looking-at: Unescaped literal `+' (pos 0)
  "+A"
   ^
8.elisp:10:4: In call to looking-at: Unescaped literal `+' (pos 0)
  "+B"
   ^
8.elisp:15:4: In call to looking-at: Unescaped literal `*' (pos 0)
  "*C"
   ^
